### DISCUSSION TITLE
getting CERTIFICATE_VERIFY_FAILED

### DISCUSSION THREAD

**Discussion by LarryMartell** [mitmproxy/mitmproxy#7018] [General]
I am on a Mac, I followed all the instructions to install the mitmproxy CA certificate, and set up the proxies. When my app sends out a request it fails with [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)

And ideas on how to get past this?

**Comment by emanuele-em** [comment]
maybe a stupid question: a part from store the certificate into the keychain, did you manually trust it?
